tethys_cli/scaffold_commands.py

```diff
@@ -116,7 +116,7 @@ def render_path(path, context):
     return path
 
 
-def scaffold_command(args):
+def scaffold_command(args, unknown_args=None):
```

Comment:
What happens when you provide an unknown argument now? Seems like it wouldn't catch it in the commands that don't use it. Perhaps good default behavior would be to combine these where `unknown_args` isn't used:

```
def generate_command(args, unknown_args=""):
    if unknown_args is not None:
        args = args + unknown_args
    ...
```